org.musicbrainz.Picard.appdata.xml

```diff
@@ -1,20 +1,57 @@
+  <id>org.musicbrainz.Picard</id>
+  <project_license>GPL-2.0+</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
   <name>Picard</name>
   <summary>MusicBrainz's music tagger</summary>
+
   <description>
     <p>
       Tag your music with the next generation MusicBrainz tagger.
     </p>
   </description>
-  <url type="homepage">https://picard.musicbrainz.org/</url>
+
+  <launchable type="desktop-id">picard.desktop</launchable>
+
   <screenshots>
     <screenshot type="default">https://www.daitauha.fr/static/screenshots/picard/main.png</screenshot>
   </screenshots>
-  <project_license>GPL-2.0+</project_license>
-  <metadata_license>CC0-1.0</metadata_license>
-  <!-- FIXME: change this to an upstream email address for spec updates
-  <update_contact>someone_who_cares@upstream_project.org</update_contact>
-  -->
-</application>
+
+  <url type="homepage">https://picard.musicbrainz.org/</url>
+
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
+  </content_rating>
+
+  <releases>
+    <release date="2017-06-20" version="1.4.2"/>
+    <release date="2017-04-15" version="1.4.1"/>
```

Comment:
Some descriptions would be nice here, but if you can't find anything quickly from upstream don't bother, we can do that better for 2.0. :wink:

Reply:
It's a little cumbersome to adapt official Picard's changelog sadly :(

Reply:
Yeah, no worries. Having the versions is already good, it improves dramatically what users see in Software.